test/parallel/test-stream2-readable-wrap.js

```diff
@@ -1,24 +1,21 @@
 'use strict';
 require('../common');
```

Comment:
`common` is undefined because this line has not been changed to `const common = require('../common');` So again, there's no way this test can actually run as is.